### PR TITLE
Remove -a from go build

### PR DESCRIPTION
**Pull request by AaronFriel**

### CODE REVIEW

**Review by excavador — APPROVED** [review]

**Review by excavador — APPROVED** [review]

**Comment by AaronFriel** [comment]
@stack72 Same as other comments - No option on sidebar to add you as a reviewer, you have merge rights here, right? Could you check if engineers have write permissions to the repo?

**Comment by excavador** [comment]
@AaronFriel yes, I can't merge this pull request :(
@stack72 could you please check my permissions to the repo?

**Comment by stack72** [comment]
same as other provider

**Pull request merged**
